stellarphot/core.py

```diff
@@ -1217,7 +1217,9 @@ def from_vizier(
 
         return cat
 
-    def passband_columns(self, passbands=None, transformer=None):
+    def passband_columns(
+        self, passbands=None, transformer=None, transformer_kwargs=None
+    ):
         """
         Return an `astropy.table.Table` with passbands as column names instead
```

Comment:
You might want to document `transformer_kwargs` as a parameter in the docstring (I can guess from context, but just for completeness).